test/BKTClient.spec.ts

```diff
@@ -31,7 +31,7 @@ import {
   TestPlatformModule,
   FakeClock,
 } from './utils'
-import fetch from 'cross-fetch'
+// import fetch from 'cross-fetch'
```

Comment:
`cross-fetch` is outdated and isn't getting any update more than a years.

since Fetch is widely available (at least from Node.js 18 and higher).
so we could get rid of "cross-fetch" as a dependency.

I am testing more to make sure. So far all tests passed when run on both browser & node environment.

Reply:
I tested with node 18 and higher and it seems good.

mswjs example for `vitest` could run on both browser  & node environment without `cross-fetch`
https://github.com/mswjs/examples/tree/main/examples/with-vitest